shop/tests/test_views.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase, Client
+from django.urls import reverse
+from shop.models import CustomUser, Image, Item, Order, Product,Category, ProductSize, Size, Comment, Sale, SaleProduct
+from ..utils.constant import LENGTH_PAGE, STATUS_ORDER, FILTER, SALE
+
+class TestView(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        test_user1 = CustomUser.objects.create_user(email= 'test1@gmail.com', username='testuser1', password='1X<ISRUkw+tuK')
+        test_user2 = CustomUser.objects.create_user(email= 'test2@gmail.com', username='testuser2', password='2HJ1vRV0Z&3iD')
+        category = Category.objects.create(name="xyz", description="xyz")
```

Comment:
data test cũng cần đẹp đẽ em hè, lấy dữ liệu cho chuẩn nhá
vd: `name="Category test 1", description="Description for category test 1"`